NuRadioReco/framework/event.py

```diff
@@ -688,4 +688,5 @@ def deserialize(self, data_pkl):
         if "__modules_event" in data:
             self.__modules_event = data['__modules_event']
         if "__modules_station" in data:
-            self.__modules_station = data['__modules_station']
+            for key, value in data['__modules_station'].items():
+                self.__modules_station[key] = value
```

Comment:
why not `self.__modules_station[key].update(data['__modules_station'])`?

Reply:
The benefit is not out weighing the effort

Reply:
Sorry, maybe I got my tone wrong on that response - the answer was no reason, other than I wasn't 100% sure how `dict.update` works. Having read the documentation it would be equivalent to what I wrote, so I can in principle replace it still. I think I got it confused in my mind with a pandas `update` method that contrary to my expectation did not overwrite existing keys (in my memory, at least).